### ISSUE TITLE
Refactoring tests to reduce test code duplication

### ISSUE THREAD

**Issue by Sharaal** [Sharaal/sql-pg#19]
Currently most of the tests have the same behavior:
- Create the actual sql/tag helper object
- In case of sql object assert the text and parameter against expected ones and check the symbol
- In case of tag helper assert the text, parameter and symbol are undefined
- Call the actual as function to test the usage as tag helper with different parameter position

This are a lot of redundant test code to check that things in every test case.

It would be nice to have some test functions to wrap the test steps, e.g.:

```javascript 
const value1 = 'value1'
const value2 = 'value2'
const value3 = 'value3'
testSql(
  sql`SELECT "*" FROM "table" WHERE "column1" = ${value1} AND "column2" = ${value2} AND "column3" = ${value3}`,
  {
    text: {
      0: 'SELECT "*" FROM "table" WHERE "column1" = $1 AND "column2" = $2 AND "column3" = $3',
      5: 'SELECT "*" FROM "table" WHERE "column1" = $6 AND "column2" = $7 AND "column3" = $8'
    },
    parameters: ['value1', 'value2', 'value3']
  }
)
```

And similar for the tag helpers, e.g.:

```javascript
testTagHelper(
  sql.values(['value1', 'value2', 'value3']),
  {
    text: {
      0: '$1, $2, $3',
      5: '$6, $7, $8'
    },
    parameters: ['value1', 'value2', 'value3']
  }
)
```

It also increase the readability of the test cases.

**Comment by Sharaal** [comment]
PR: https://github.com/Sharaal/sql-pg/pull/20

**Comment by Sharaal** [comment]
Released with v5.1.4
